GOVERNANCE.md

```diff
@@ -1,3 +1,187 @@
+* [Daniel Fedorin](https://github.com/DanilaFe)
+* [Michael Ferguson](https://github.com/mppf)
+* [Abhishek Girish](https://github.com/agirish)
+* [Shreyas Khandekar](https://github.com/ShreyasKhandekar)
+* [Vass Litvinov](https://github.com/vasslitvinov)
+* [David Longnecker](https://github.com/dlongnecke-cray)
+* [Ben McDonald](https://github.com/bmcdonald3)
+* [Ahmad Rezaii](https://github.com/arezaii)
+* [Anna Rift](https://github.com/riftEmber)
+* [Elliot Ronaghan](https://github.com/ronawho)
+* [Michelle Strout](https://github.com/mstrout)
+* [Karlon West](https://github.com/karlonw)
+
+
+Decision-Making
+---------------
+
+Generally speaking, decision-making on the Chapel project is done by
+the technical leadership and committers in this section on a consensus
+basis, taking the opinions of end-users into account when the decision
+is likely to affect them.  Project-wide decisions tend to involve all
+technical leads and committers, while those that are specific to a
+given effort can be made by a subteam lead and its members, keeping
+the tech lead apprised of key decisions.
+
+Votes are rarely taken in a binding manner, but are often used to
+gauge the level of support for an idea or direction.  In such votes,
+those in the minority are typically given the opportunity to continue
+to push for alternative approaches, or to concede to the majority,
+having registered their opinion.
+
+For non-trivial decisions in language design, library APIs, or code
+architecture, the project or subteam typically forms an ad hoc subteam
+including at least three committers to explore the topic and propose a
+solution, where anyone on the broader team is encouraged to
+participate if they feel interested or invested.  Once the ad hoc
+subteam has reached a conclusion, it is summarized back to the full
+team.  Generally speaking, those who did not join the ad hoc subteam
+are expected to go along with its decision, unless they have an
+insight or strong reason that the proposal is untenable—particularly
+if it involves factors the subteam was not, or could not have been,
+aware of.  In practice, this has happened only very rarely.
+
+Decisions that relate to the design of the language or libraries are
+captured and discussed on public [GitHub
+issues](https://github.com/chapel-lang/chapel/issues?q=is%3Aissue) as
+a means of soliciting input from the community. Many of these issues
+are opened or requested by users directly. Decisions that are
+particularly impactful to user codes are advertised directly to users
+to get their feedback by pointing them to the GitHub issues or
+contacting them directly.  For some of the most impactful changes, we
+have run polls and worked hard to involve as many users as possible in
+order to avoid surprises and attempt to develop a consensus community
+view.  In extreme cases, the tech lead may veto a decision, though it
+is expected that this would be used only very rarely.  They may also
```

Comment:
I prefer "The technical lead may also" (I started reading the sentence thinking that "They" was referring to "extreme cases".)